utils/src/main/java/software/amazon/awssdk/utils/Validate.java

```diff
@@ -47,7 +46,7 @@
  * <p>#ThreadSafe#</p>
  * @see java.lang.String#format(String, Object...)
  */
-@ReviewBeforeRelease("Remove the methods we don't end up using (and software.amazon.awssdk.core.util.ValidationUtils).")
+//TODO: Remove the methods we don't end up using (and software.amazon.awssdk.core.util.ValidationUtils).
```

Comment:
There is one unused method. Can we remove it now.

Reply:
Yeah, will do